### DISCUSSION TITLE
Fix the blender for video stitching.

### DISCUSSION THREAD

**Discussion by JoelDTerry** [OpenStitching/stitching#217] [General]
I'm stitching two videos together following a previous post about  re-using the homography matrix, however the blend is not consistent leading to a stutter effect where the two images overlap. 

Is it possible to reuse a previous blend so that the overlap region is consistent ? 

Thanks in advance

**Comment by lukasalexanderweber** [comment]
I guess you can save the first obtained seam masks as its done for the cameras and reuse them in every frame
